### ISSUE TITLE
random number next to the js file?

### ISSUE THREAD

**Issue by mxriverlynn** [kendo-labs/kendo-bootstrapper#11]
![](http://cl.ly/image/34072X3H3z3x/Screen%20Shot%202013-07-02%20at%209.41.13%20PM.png)

not sure what this number next to the file is... there's no label and no tooltip / alt text to tell me
